src/types.ts

```diff
@@ -56,6 +56,7 @@ export type GraphQlQueryResponse<ResponseData> = {
   data: ResponseData;
   errors?: [
     {
+      type: string;
```

Comment:
Let's add a reference to the pull request

```suggestion
      // https://github.com/octokit/graphql.js/pull/314
      type: string;
```

Reply:
that sounds good 👌